Index: dahdi-tools-2.10.0/Makefile
===================================================================
--- dahdi-tools-2.10.0.orig/Makefile
+++ dahdi-tools-2.10.0/Makefile
@@ -240,7 +240,6 @@ ifneq (,$(BINS))
 endif
 ifeq (,$(wildcard $(DESTDIR)$(CONFIG_FILE)))
 	$(INSTALL) -d $(DESTDIR)$(CONFIG_DIR)
-	$(INSTALL) -m 644 system.conf.sample $(DESTDIR)$(CONFIG_FILE)
 endif
 	install -d $(DESTDIR)$(DATA_DIR)
 	tar cf - -C hotplug $(ASSIGNED_DATA_SCRIPTS) | tar xf - -C $(DESTDIR)$(DATA_DIR)/
